pages/index.html

```diff
@@ -2,15 +2,15 @@
 
 {% load i18n %}
 
-{% block title %}{% trans "Free MP3 DJ Mixing Software" %}{% endblock %}
+{% block title %}{% trans "Free Digital DJ MP3 Mixing Software" %}{% endblock %}
```

Comment:
maybe we can drop "mp3" now?  It seems sort of quaint at this point.  (optional)